package.json

```diff
@@ -28,6 +28,7 @@
     "gulp-mocha": "^4.3.1",
     "gulp-nodemon": "^2.2.1",
     "gulp-sourcemaps": "^1.6.0",
+    "gulp-typescript": "^3.2.2",
```

Comment:
Where do we use this module?

Reply:
This module does not use in this patch, it will use in next patch which for "typescript apply in server"